app/repositories/password_reset_tokens.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+
+from pydantic import BaseModel
+
+import app.state
+
+
+class PasswordResetToken(BaseModel):
+    id: int
+    hashed_token: str
+    username: str
+    created_at: datetime
+
+
+READ_PARAMS = """\
+    id, k, u, t
+"""
+
+
+async def create(*, username: str, hashed_token: str) -> PasswordResetToken:
+    query = """\
+        INSERT INTO password_recovery (k, u, t)
+        VALUES (:hashed_token, :username, NOW())
```

Comment:
```suggestion
        INSERT INTO password_recovery (k, u)
        VALUES (:hashed_token, :username)
```

Reply:
this does have a default of CURRENT_TIMESTAMP

Reply:
yes